cxx/assets/test_files/test_crosscat_generator.py

```diff
@@ -0,0 +1,87 @@
+# Generates a 4 column dataset for one data type.
+import numpy as np
+
+NUM_SAMPLES_1 = 33
+NUM_SAMPLES_2 = 50
+NUM_SAMPLES = 100
+
```

Comment:
What do you think about adding some tests for this program?

Reply:
I'm adding integration tests in a later PR that will test the files (test some basic invariants, like creating two IRMs, etc)